### ISSUE TITLE
connect_get_namespaced_pod_exec stdout incorrect default in documentation

### ISSUE THREAD

**Issue by mirskiy** [kubernetes-client/python#1568]
Per the documentation for [`connect_get_namespaced_pod_exec`](https://raw.githubusercontent.com/kubernetes-client/python/master/kubernetes/docs/CoreV1Api.md#connect_get_namespaced_pod_exec):  

> stdout: Redirect the standard output stream of the pod for this call. Defaults to **true**

In testing, it defaults to **false**

**What happened (please include outputs or screenshots)**:
When running `connect_get_namespaced_pod_exec` without `stdout` kwarg set, no output is returned.  
Further, if none of `stdin`, `stdout`, or `stderr` are set, `connect_get_namespaced_pod_exec` returns a 400 error. This is inconsistent with these parameters being "optional"

**What you expected to happen**:
`stdout` should default to true and I should see output from the command.

**How to reproduce it (as minimally and precisely as possible)**:
```
config.load_kube_config()
v1 = client.CoreV1Api()
name = "busybox-test"
exec_command = ['/bin/sh', '-c', 'echo Test']

resp = stream(v1.connect_get_namespaced_pod_exec, name, 'default', command=exec_command,
          stderr=True, stdout=True)
print("Response 1: ")
print(resp)
print("-----")

resp = stream(v1.connect_get_namespaced_pod_exec, name, 'default', command=exec_command,
          stderr=True)
print("Response 2: ")
print(resp)
print("-----")

resp = stream(v1.connect_get_namespaced_pod_exec, name, 'default', command=exec_command)
print("Response 3: ")
print(resp)
print("-----")
```

Output:  
```
Response 1:
Test

-----
Response 2:

-----
Traceback (most recent call last):
  File "/home/dan/robo/venv/lib64/python3.6/site-packages/kubernetes/stream/ws_client.py", line 468, in websocket_call
    client = WSClient(configuration, url, headers, capture_all)
  File "/home/dan/robo/venv/lib64/python3.6/site-packages/kubernetes/stream/ws_client.py", line 62, in __init__
    self.sock = create_websocket(configuration, url, headers)
  File "/home/dan/robo/venv/lib64/python3.6/site-packages/kubernetes/stream/ws_client.py", line 452, in create_websocket
    websocket.connect(url, header=header)
  File "/home/dan/robo/venv/lib64/python3.6/site-packages/websocket/_core.py", line 253, in connect
    self.handshake_response = handshake(self.sock, *addrs, **options)
  File "/home/dan/robo/venv/lib64/python3.6/site-packages/websocket/_handshake.py", line 57, in handshake
    status, resp = _get_resp_headers(sock)
  File "/home/dan/robo/venv/lib64/python3.6/site-packages/websocket/_handshake.py", line 143, in _get_resp_headers
    raise WebSocketBadStatusException("Handshake status %d %s", status, status_message, resp_headers)
websocket._exceptions.WebSocketBadStatusException: Handshake status 400 Bad Request
...
```

**Anything else we need to know?**:
I know this has to do with the kubernetes OpenAPI spec, but unfortunately I don't understand the kubernetes source well enough to get any further or report the issue there.

**Environment**:
- Kubernetes version (`kubectl version`): 
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}                                                                                                                 Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.15", GitCommit:"58178e7f7aab455bc8de88d3bdd314b64141e7ee", GitTreeState:"clean", BuildDate:"2021-09-15T19:18:00Z", GoVersion:"go1.15.15", Compiler:"gc", Platform:"linux/amd64"}  
```
- OS (e.g., MacOS 10.13.6):    CentOS
- Python version (`python --version`)   Python 3.6.8  
- Python client version (`pip list | grep kubernetes`)   19.15.0a1

**Comment by roycaihw** [comment]
@mirskiy Nice finding! and sorry for the confusion. It seems that the defaulting behavior was changed in 1.7 (https://github.com/kubernetes/kubernetes/issues/49586, https://github.com/kubernetes/kubernetes/pull/49579), but we didn't update the documentation. 

Given the behavior has existed for a long time, I'd suggest we fix the k/k documentation. The fix needs to be made to the [types.go file](https://github.com/kubernetes/kubernetes/blob/a53e2eaeaba064309dceca2dc27f3ac09c6375b0/staging/src/k8s.io/api/core/v1/types.go#L5297), and you will need to run `make update` to re-generate the OpenAPI spec. Let me know if you have any questions.

/unassign

**Comment by roycaihw** [comment]
Please feel free to assign yourself and give it a try

/help

**Comment by roycaihw** [comment]
@DiptoChakrabarty Awesome, thanks!

The documentation in this repo will be fixed when we generate a 1.23 client. 

/assign @DiptoChakrabarty
/close
